crypto/evp/evp_local.h

```diff
@@ -274,3 +274,9 @@ void evp_names_do_all(OSSL_PROVIDER *prov, int number,
                       void (*fn)(const char *name, void *data),
                       void *data);
 int evp_cipher_cache_constants(EVP_CIPHER *cipher);
+
+EVP_PKEY_CTX *evp_pkey_ctx_new_int(OPENSSL_CTX *libctx,
+                                   EVP_PKEY *pkey, ENGINE *e,
+                                   const char *name, const char *propquery,
+                                   int id);
```

Comment:
Also declared in `include/crypto/evp.h`...

Reply:
removed from evp_local.h